packages/block-editor/src/components/global-styles/typography-utils.js

```diff
@@ -45,24 +45,15 @@ import { getComputedFluidTypographyValue } from '../font-sizes/fluid-utils';
 export function getTypographyFontSizeValue( preset, typographySettings ) {
```

Comment:
Dumb nit but can we rename `typographySettings` here to `typographyOptions` to indicate that you're supposed to pass it the return value from `getFluidTypographyOptionsFromSettings`?

`settings` → `typographySettings` → `typographyOptions`

Reply:
Good call. Will do